src/components/ui/ProductTitle/ProductTitle.tsx

```diff
@@ -18,14 +20,14 @@ interface ProductTitleProp {
 
 function ProductTitle({ title, label, refNumber }: ProductTitleProp) {
   return (
-    <div className="product-title">
-      <div className="product-title__header">
+    <div className={styles.fsProductTitle}>
```

Comment:
```suggestion
    <header className={styles.fsProductTitle}>
```

I think we could take the opportunity to update this tag 🙇‍♀️

Reply:
Completely agree! done [here](https://github.com/vtex-sites/nextjs.store/pull/84/commits/a5af6985024f41aa176ac8c7b9ea919f0ebd5f4c)